utils/sane-backends/Makefile

```diff
@@ -134,6 +137,11 @@ $(call Package/sane-backends/Default/description)
 This package contains the SANE daemon.
 endef
 
+define Package/sane-daemon/conffiles
```

Comment:
On 16-03-24 11:56 PM, Luiz Angelo Daros de Luca wrote:
> In utils/sane-backends/Makefile
> <https://github.com/openwrt/packages/pull/2543#discussion_r57417238>:
>
>> @@ -134,6 +137,11 @@ $(call Package/sane-backends/Default/description)
>>  This package contains the SANE daemon.
>>  endef
>>
>> +define Package/sane-daemon/conffiles
>
> Why this define was moved? It was after 'define Package/sane-daemon/install'
> If you have a reason, we can keep it here but it is better to avoid
> moving code.

Ok, that was unintentional.  This was actually originally based on 
oldpackages prior to sane-backends getting merged back into packages.

Regards,

Daniel